src/x/headers/headers.go

```diff
@@ -43,6 +43,11 @@ const (
 	// Valid values are "unaggregated" or "aggregated".
 	MetricsTypeHeader = M3HeaderPrefix + "Metrics-Type"
 
+	// PromTypeHeader sets the prometheus metric type. Valid values are
+	// "counter", "gauge", etc. (see src/query/api/v1/handler/prometheus/remote/write.go
+	// field `headerToMetricType`)
+	PromTypeHeader = M3HeaderPrefix + "Prom-Type"
```

Comment:
Can we change this to be `Prometheus-Metric-Type` in case we there are other "types" we may want to specify by header in the future?